actionpack/lib/action_controller/api/api_rendering.rb

```diff
@@ -0,0 +1,14 @@
+module ActionController
+  module ApiRendering
```

Comment:
What is the difference between ApiRendering and the one included inside `ActionController::Base`?